emscripten.py

```diff
@@ -599,7 +599,7 @@ def make_emulated_param(i):
     if metadata['simdUint32x4']:
       simdinttypes += ['Uint32x4']
       simdintfloatfuncs += ['fromUint32x4Bits']
-    if metadata['simdInt32x4']:
+    if metadata['simdInt32x4'] or settings['SIMD']: # Always import Int32x4 when building with -s SIMD=1, since memcpy is SIMD optimized.
```

Comment:
can simdInt32x4 be false but SIMD be true? (if not, then the if could be simplified to just check for SIMD?)